plugins/redfish/fu-redfish-client.c

```diff
@@ -100,11 +100,69 @@ fu_redfish_client_coldplug_inventory (FuRedfishClient *self,
 				      GError **error)
 {
 	JsonArray *members;
+	JsonObject *member_id;
+	const gchar *member_uri;
```

Comment:
You can move these inside the loop, thanks.

Reply:
Ok, I'll push a refreshed patch.